### ISSUE TITLE
is there anyway for normal PIP mode for without video player for both android and iOS?

### ISSUE THREAD

**Issue by RaviGaruda** [kesmitopiwala/pip_flutter#2]

**Comment by harshmdr** [comment]
Is there any way to show our custom widget in PIP in ios?

**Comment by SethuSenthil** [comment]
> Is there any way to show our custom widget in PIP in ios?

No. iOS does not support custom views in PIP. Only the system video player is supported (as of iOS 16).

**Comment by manish3008** [comment]
can someone tell that how to integrate this normal video player in pip mode in ios?.for me its not working.
